index.js

```diff
@@ -6277,11 +6278,11 @@ export async function submitBom(args, bomContents) {
         console.log(
           "Unable to submit the SBOM to the Dependency-Track server using POST method",
         );
-        console.log(error);
       }
     } else {
       console.log("Unable to submit the SBOM to the Dependency-Track server");
-      console.log(error);
     }
+    console.log(error.response?.body);
+    return error.response;
```

Comment:
Do we need the whole response back since it might have sensitive headers?

Reply:
You'd limit only to the body?

Reply:
Done